src/common/snippets/include/snippets/lowered/pass/reduce_decomposition.hpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "pass.hpp"
+
+namespace ov {
+namespace snippets {
+namespace lowered {
+namespace pass {
+
+/**
+ * @interface ReduceDecomposition
+ * @brief Decomposes snippets::Reduce operations to a range of low-level operations on linear IR
+ * @attention Only Reduce by last dimension is supported
+ * @ingroup snippets
+ */
+class ReduceDecomposition : public RangedPass {
```

Comment:
As we discussed offline: the follow-up task is to move `ReduceDecomposition` on data-flow stage.

Reply:
The follow-up ticket is created: 134837